core/domain/user_jobs_one_off_test.py

```diff
@@ -196,6 +196,145 @@ def test_user_contributions_get_created_after_running_the_job(self):
+        user_jobs_one_off.PopulateUserAuthModelOneOffJob.enqueue(job_id)
+        self.assertEqual(
+            self.count_jobs_in_taskqueue(
+                taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
+        self.process_and_flush_pending_tasks()
+
+        stringified_output = (
+            user_jobs_one_off.PopulateUserAuthModelOneOffJob.get_output(
+                job_id))
+        output = {}
+        for stringified_distribution in stringified_output:
+            message_list = ast.literal_eval(stringified_distribution)
+            output[message_list[0]] = message_list[1]
+        return output
+
+    def setUp(self):
+        super(PopulateUserAuthModelOneOffJobTests, self).setUp()
+        self.user_a_model = user_models.UserSettingsModel(
+            id=self.USER_A_ID,
+            gae_id=self.USER_A_GAE_ID,
+            email=self.USER_A_EMAIL,
+        )
+        self.user_b_model = user_models.UserSettingsModel(
+            id=self.USER_B_ID,
+            gae_id=self.USER_B_GAE_ID,
+            email=self.USER_B_EMAIL
+        )
+        self.user_a_model.put()
+
+    def test_before_migration_user_does_not_exist_in_user_auth_model(self):
+        self.assertIsNone(user_models.UserAuthModel.get_by_id(self.USER_A_ID))
+
+    def test_one_off_job_migrate_single_new_user_successfully(self):
+        self.assertIsNone(user_models.UserAuthModel.get_by_id(self.USER_A_ID))
+
+        output = self._run_one_off_job()
+
+        user_auth_model = user_models.UserAuthModel.get_by_auth_id(
+            feconf.AUTH_METHOD_GAE, self.USER_A_GAE_ID)
+        self.assertEqual(user_auth_model.id, self.USER_A_ID)
+
+        expected_output = {
+            'Created': [
+                {'user_id': self.USER_A_ID, 'gae_id': self.USER_A_GAE_ID}]
+        }
+        self.assertEqual(output, expected_output)
+
+    def test_one_off_job_migrate_multiple_new_users_successfully(self):
+        self.user_b_model.put()
+        self.assertIsNone(user_models.UserAuthModel.get_by_id(self.USER_A_ID))
+
+        output = self._run_one_off_job()
+
+        user_auth_model = user_models.UserAuthModel.get_by_auth_id(
+            feconf.AUTH_METHOD_GAE, self.USER_A_GAE_ID)
+        self.assertEqual(user_auth_model.id, self.USER_A_ID)
+        user_auth_model = user_models.UserAuthModel.get_by_auth_id(
+            feconf.AUTH_METHOD_GAE, self.USER_B_GAE_ID)
+        self.assertEqual(user_auth_model.id, self.USER_B_ID)
+
+        expected_output = [
+            {'user_id': self.USER_A_ID, 'gae_id': self.USER_A_GAE_ID},
+            {'user_id': self.USER_B_ID, 'gae_id': self.USER_B_GAE_ID},
+            ]
+
+        # To ensure both lists have same elements.
+        for user in expected_output:
+            self.assertIn(user, output['Created'])
+        for user in output['Created']:
+            self.assertIn(user, expected_output)
+        self.assertEqual(['Created'], output.keys()) # pylint: disable=dict-keys-not-iterating
```

Comment:
```suggestion
        self.assertItemsEqual(output['Created'], expected_output)
        self.assertItemsEqual(['Created'], output.keys())
```

Reply:
Maybe even just 
```suggestion
        self.assertItemsEqual(output, expected_output)
```
might work if you modify the `expected_output`

Reply:
Used a different way, but removed the extra code.